static_frame/core/frame.py

```diff
@@ -3006,6 +3007,18 @@ def __deepcopy__(self, memo: tp.Dict[int, tp.Any]) -> 'Frame':
     #     '''
     #     return self.__copy__() #type: ignore
 
+    def __sizeof__(self: 'Frame') -> int:
+        # TODO: Check if we need to count weakref
+        # TODO: Make sure these don't have duplicates
+        # TODO: Consider skipping counting arrays used in multiple nested objects
+        return sum(getsizeof(o) for o in [
+            self.__weakref__,
```

Comment:
I'm not sure we need to get the size of `__weakref__`. See: https://stackoverflow.com/a/36788452/7615851

What are your thoughts?

Reply:
Reading that and thinking more into it, I'm against counting the __weakref__ since it's not related to the memory needed for the object to be fully functional.